client/resources/original_messages.json

```diff
@@ -75,6 +75,14 @@
     "description": "The title of a message to the user that tells them Outline will keep connected unless explicitly told to disconnect. Outline is the product name and can be found in the translation glossary.",
     "message": "Stay protected, always"
   },
+  "basic_access_content": {
+    "description": "The body text of a dialog box explaining what 'Basic Access' means for a server connection.",
+    "message": "Outline Basic Access is an alternative way to connect to the open internet. It offers limited protection of your internet traffic. This approach will make it harder for outside observers to see what you connect to online, without the use of a VPN (proxy) server.<br><br>This feature may help you connect to the open internet if you do not have a traditional Outline VPN access key, or if your VPN access key is not successfully connecting. <a src='TODO'>Learn more</a>"
+  },
+  "basic_access_title": {
+    "description": "The title of a dialog box explaining what 'Basic Access' means for a server connection.",
+    "message": "Basic Access<br><span style='color:hsl(176, 97%, 26%);'>A New Way to Get Online</span>"
```

Comment:
Can we use css variables here? Though mixing the CSS style into texts seems a little bit weird. Maybe we can separate the title into two texts?